include/ccmath/math/fmanip/ldexp.hpp

```diff
@@ -8,7 +8,69 @@
 
 #pragma once
 
+#include <ccmath/internal/support/bits.hpp>
+#include "ccmath/internal/support/floating_point_traits.hpp" //this is included in bits, but i'm going to include it anyways
+
 namespace ccm
 {
+	// this function multiples parameter x by two to the power of paramter powerOf2
```

Comment:
Use doxygen for commenting the intent of a function. There is examples of what this look like inside of the basic module. You can also use https://en.cppreference.com/w/cpp/numeric/math/ldexp as a guide for how you should describe the functionality of the function and its values and such.

Reply:
Also make sure you are applying doxygen comments to all of the specializations also!